runtime/gc_glue_java/ScavengerBackOutScanner.cpp

```diff
@@ -328,4 +328,48 @@ MM_ScavengerBackOutScanner::backoutUnfinalizedObjects(MM_EnvironmentStandard *en
 	env->getGCEnvironment()->_unfinalizedObjectBuffer->flush(env);
 }
 #endif /* J9VM_GC_FINALIZATION */
+
+/**
+ * Backout the continuation objects.
+ * Move continuation backout processing in scanAllSlots(), scavenger abort would never happen after continuationObjectList processing
+ * (so only need to backout list._head from _priorHead).
+ * Here walk the lists in the Evacuate region only for helping to back out the references in java stacks of the unmounted continuations.
+ */
+void
+MM_ScavengerBackOutScanner::backoutContinuationObjects(MM_EnvironmentStandard *env)
+{
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+	if (_extensions->isConcurrentScavengerEnabled()) {
+		/* nothing need to do for concurrent scavenger case */
```

Comment:
rather than just stating the obvious, try to explain why.
something like: for CS no need to backout stack references, since they will be fixed up to point to the new version of the object (if not already do so), later during marking when continuation objects are found live